pkg/daemon/update.go

```diff
@@ -702,6 +692,36 @@ func (dn *Daemon) updateOS(config *mcfgv1.MachineConfig) error {
 	return nil
 }
 
+func (dn *Daemon) readPendingConfig() (string, error) {
+	// TODO(runcom): msgid has been generated with journalctl --new-id128, move it to a const
+	journalOutput, err := exec.Command("journalctl", "-b", "-1", "-o", "cat", "MESSAGE_ID=34c7912c5dd2454286097f8f92a22e9e").CombinedOutput()
```

Comment:
output to json to read BOOT_ID and MESSAGE

Reply:
also, we might want to have a customized "machine-config-daemon-pending-config" as msgid - not sure about opinions on this?